### DISCUSSION TITLE
Is there any way to bundle proxy to tauri?

### DISCUSSION THREAD

**Discussion by adoin** [tauri-apps/tauri#11902] [Q&A]
I was asked to implement a platform. Which customers view websites there.
![image](https://github.com/user-attachments/assets/7f3a4848-9f1f-4b26-a618-b9ddf720dc23)

**Comment by adoin** [comment]
If tauri is not able to do this. can any body tell me which tech can help me?

**Comment by xuchaoqian** [comment]
@adoin  This API is for you: [proxy_url](https://docs.rs/tauri/2.1.1/tauri/webview/struct.WebviewBuilder.html#method.proxy_url)

**Reply by adoin** [reply]
> @adoin This API is for you: [proxy_url](https://docs.rs/tauri/2.1.1/tauri/webview/struct.WebviewBuilder.html#method.proxy_url)

Thank your for reply. 
Another requirement is get the inputs or token inside webview, is this possible?

**Reply by xuchaoqian** [reply]
@adoin You could use [webview.eval](https://docs.rs/tauri/2.1.1/tauri/webview/struct.Webview.html#method.eval) to inject any logic into a web page to perform any action.

**Reply by adoin** [reply]
> @adoin You could use [webview.eval](https://docs.rs/tauri/2.1.1/tauri/webview/struct.Webview.html#method.eval) to inject any logic into a web page to perform any action.

Looks good, thank you.
